packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -350,14 +350,16 @@ export default class Tile extends Surface {
   }
   /* ------------------------------ Progress Bar ------------------------------ */
 
-  get _progressBarHeight() {
+  get _progressBarY() {
     return (
       (this._ProgressBar &&
-        this._ProgressBar._getTransition('h')._targetValue +
-          this.style.paddingY) ||
-      0
+      this._ProgressBar._getTransition('alpha')._targetValue !== 0
+        ? this._ProgressBar._getTransition('y')._targetValue ||
+          this._ProgressBar.y
+        : 0) || 0
```

Comment:
could we comment this? I'm just trying to read it and barely able to wrap my head around all the nested-ness

Reply:
ha for sure 😅 updated with a big comment block, hopefully it helps!